doc/site/core/IO.markdown

```diff
@@ -0,0 +1,44 @@
+^title io Class
```

Comment:
I'm pretty sure the lowercasing was just about the filenames, not this title. :-)